src/twisted/runner/test/test_procmon.py

```diff
@@ -130,6 +131,28 @@ def setUp(self):
         self.pm.threshold = 10
 
 
+    def test_reprLooksGood(self):
+        """
+        Repr includes all details
+        """
+        self.pm.addProcess("foo", ["arg1", "arg2"],
+                           uid=1, gid=2, env={})
+        representation = repr(self.pm)
+        self.assertIn('foo', representation)
+        self.assertIn('1', representation)
+        self.assertIn('2', representation)
+
+
+    def test_simpleReprLooksGood(self):
+        """
+        Repr does not include unneeded details
```

Comment:
How can I tell which are unneeded details?

There is no full stop. Is this doctring complete or we should expect more information :)

Together with test_reprLooksGood the docstring of these test don't add to much information on top of what is communicated by the test names.

I don't understand why are you checkin for paranthesis in the repr() result.